modules/schematics/src/feature/index.ts

```diff
@@ -16,7 +16,7 @@ export default function(options: FeatureOptions): Rule {
         name: options.name,
         path: options.path,
         project: options.project,
-        spec: false,
+        skipTest: true,
```

Comment:
Not for now, but could we introduce a `skipTest` flag for the `feature` schematic @brandonroberts ?
By default, we create spec files for every schematic, but not for this schematic.